tests/query/v1/test_go.py

```diff
@@ -1031,24 +1031,33 @@ def test_reversely_with_pipe(self):
         expected_data = {
             "column_names" : ["$^.team.name", "$$.player.name"],
             "rows" : [
-                ["Cavaliers", "Kyrie Irving"],
```

Comment:
Why change this?

Reply:
rank is missing from the  previous test case，after add rank field,  output of the statement before pipe are two 'Cavaliers'